File: taskmanager/app/entities/Project.go
package entities

type Project struct {
	Id              int    ` json:"id" xml:"id" `
	Name            string ` json:"name" xml:"name" `
	CreatorId       int    ` json:"creatorId" xml:"creatorId" `
	AimOfTheProject string ` json:"aimOfTheProject" xml:"aimOfTheProject" `
}

File: taskmanager/public/js/src/entities/Project.js
'use strict'

class Project {

    #id;
    #name;
    #creatorId;
    #aimOfTheProject


    constructor(id, name, creatorId, aimOfTheProject) {

        this.#id = id;
        this.#name = name;
        this.#creatorId = creatorId;
        this.#aimOfTheProject = aimOfTheProject;
    }

    getProjectInfo() {


    }

    getName() {
        return this.#name;
    }

    setName(name) {
        this.#name = name;
    }

    getAimOfTheProject() {
        return this.#aimOfTheProject;
    }

    setAimOfTheProject(aim) {
        this.#aimOfTheProject = aim;
    }

    getId() {
        return this.#id;
    }

    getCreatorId() {
        return this.#creatorId;
    }
}